testnet/lighthouse/lighthouse.toml

```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
 listen-address = "0.0.0.0"
 eth1 = true
 http = true
-http-address = "0.0.0.0"
+http-address = "127.0.0.1"
 http-port = 4000
 enr-udp-port = 9000
 enr-tcp-port = 9000
 discovery-port = 9000
 enable-private-discovery = true
 enr-address = "0.0.0.0"
 metrics = true
-metrics-address = "0.0.0.0"
+metrics-address = "127.0.0.1"
 metrics-allow-origin = "*"
 metrics-port = 5054
-boot-nodes = 'enr:-MK4QHfHBBWZqSIxumgMmdC3zd7QyJD2shEXBhSKbay3aYwdAHrhb3lksHStJPk1tLoFOPaU4ELtaahRGUUpfm5tmhCGAYfiNvG0h2F0dG5ldHOIAAAAAAAAAACEZXRoMpBWKoojQgEABCoAAAAAAAAAgmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQOLhdJYX4WgjYXPw6GpCkuxnNWvM0KE_aeYzek95q2f-IhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A'
\ No newline at end of file
+boot-nodes = 'enr:-MK4QPAebhGrV0IdXCaVQ33cL-5oySUvBXPkXZTHhCDrqMOtITlhab5B6PL9hfQY5NuRF5ctLcSFhHMCdLT4RWxxo-SGAYiVi7DIh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCvClM1QgEABP__________gmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQISdsDnr_lxpAXRQmeOu-F_eAtxypZI4ctFavqdagbka4hzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A'
```

Comment:
Please update `prysm.yml` testnet ENR also. There are two, one needs to be changed. `enr:-MK4QHfHBBWZqSIxumgMmdC3zd7QyJD2shEXBhSKbay3aYwdAHrhb3lksHStJPk1tLoFOPaU4ELtaahRGUUpfm5tmhCGAYfiNvG0h2F0dG5ldHOIAAAAAAAAAACEZXRoMpBWKoojQgEABCoAAAAAAAAAgmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQOLhdJYX4WgjYXPw6GpCkuxnNWvM0KE_aeYzek95q2f-IhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A` is present in `prysm.yml`, change to: `enr:-MK4QPAebhGrV0IdXCaVQ33cL-5oySUvBXPkXZTHhCDrqMOtITlhab5B6PL9hfQY5NuRF5ctLcSFhHMCdLT4RWxxo-SGAYiVi7DIh2F0dG5ldHOIAAAAAAAAAACEZXRoMpCvClM1QgEABP__________gmlkgnY0gmlwhCKNxNSJc2VjcDI1NmsxoQISdsDnr_lxpAXRQmeOu-F_eAtxypZI4ctFavqdagbka4hzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`

Reply:
Done